operator/internal/manifests/internal/config/loki-config.yaml

```diff
@@ -44,6 +44,7 @@ common:
       region_name: {{ .Region }}
       container_name: {{ .Container }}
     {{- end }}
+  compactor_address: {{ .Compactor.FQDN }}:{{ .Compactor.Port }}
```

Comment:
I'm getting errors on `query-frontend` and `querier` complaining that the compactor address is malformed. I think it should be a proper URL not just `host:port`.